app/tests/test_app.py

```diff
@@ -1,5 +1,6 @@
 import unittest
 from app.app import app
+import json
 
 
 class TestApp(unittest.TestCase):
```

Comment:
I think as these classes get bigger, we should think about splitting the different endpoints into different test classes just to keep the structure easier to perceive, and we'll be able to see where we're missing coverage a lot easier